packages/jupyter-ai/src/components/settings/use-server-info.ts

```diff
@@ -83,11 +85,13 @@ export function useServerInfo(): ServerInfo {
         chat: {
           lmProvider,
           emProvider,
-          lmLocalId
+          lmLocalId,
+          emLocalId
         },
         completions: {
           lmProvider: cLmProvider,
-          lmLocalId: cLmLocalId
+          lmLocalId: cLmLocalId,
+          emLocalId: emLocalId
```

Comment:
The `completions` key only needs to store the model ID of the completions model, so it does not need a `emLocalId` key within it. This line can be dropped.

Reply:
Oh, I see that this was done to prevent a TypeScript build error. You can still delete this line. Then, change line 29 from:

```
completions: Omit<ProvidersInfo, 'emProvider'>;
```

to:

```
completions: Omit<ProvidersInfo, 'emProvider' | 'emLocalId'>;
```

This tells TypeScript that `config.completions` lacks the `emProvider` and `emLocalId` keys, which should fix the build error you faced.